### PR TITLE
better documentation: README and YARD doc

### PR DESCRIPTION
**Pull request by norydev**
# About 

Better documentation:
- better README
- YARD doc
- Better error if no client_id and client_secret in Config

For yard doc in dev mode:

```sh
gem install yard
gem install redcarpet
yard server --reload  
```

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Joren De Groof
```
Comment (joren) [inline LICENSE.txt:3]: 😅

File: README.md [hunk 2]
```diff
@@ -1,43 +1,155 @@
...
 
-TODO: Delete this and the text above, and describe your gem
+Cubscout is heavily inspired by [girlscout](https://github.com/omise/girlscout), another unofficial Ruby Client for Helpscout API V1.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+Add `gem 'cubscout'` to your application's Gemfile, and then run `bundle` to install.
+
+Or install it yourself as `gem install cubscout`.
```
Comment (joren) [inline README.md:11]: Then we should already push it to rubygems right?

**Review by joren — APPROVED** [review]
Great work 💯

File: README.md (hunk 2 above)
Comment (norydev) [reply README.md:11]: Well, it already says so in the current readme and it's not true. I wouldn't rush though 🙈

File: LICENSE.txt (hunk 1 above)
Comment (norydev) [reply LICENSE.txt:3]: Do you want to keep the copyright? 🙂

File: README.md (hunk 2 above)
Comment (joren) [reply README.md:11]: Ah, indeed. I agree.

File: LICENSE.txt (hunk 1 above)
Comment (joren) [reply LICENSE.txt:3]: No no, this is ok. Funny we didn't change it immediately, same goes for all the url's in the readme to the CoC etc.

**Pull request merged**